pkg/cmd/util/docker/docker.go

```diff
@@ -28,14 +28,16 @@ func (_ *Helper) InstallFlags(flags *pflag.FlagSet) {
 
 // GetClient returns a valid Docker client, the address of the client, or an error
 // if the client couldn't be created.
-func (_ *Helper) GetClient() (client *docker.Client, endpoint string, err error) {
-	client, err = docker.NewClientFromEnv()
-	if len(os.Getenv("DOCKER_HOST")) > 0 {
-		endpoint = os.Getenv("DOCKER_HOST")
-	} else {
+func (_ *Helper) GetClient() (*docker.Client, string, error) {
+	endpoint := os.Getenv("DOCKER_HOST")
```

Comment:
I just added the ping to the daemon as it turns out every caller of `GetClient` is doing exactly that as a follow-up step. The rest is a refactoring. BUT I had this for my own pausing of the deployer containers, since we won't do that now, I am removing this change.